pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -71,7 +72,6 @@
 
     <!-- Stable packages shipped already for netcoreapp2.0 -->
     <LibraryPackage Include="System.Data.SqlClient" />
```

Comment:
Is there a reason to keep SqlClient out now?

Reply:
Maybe not, I’ll have to double check the closure, if I remember correctly we kept it out because it was pulling System.Memory into the closure.